test/modules/core/shaderlib/project/project-glsl.spec.js

```diff
@@ -258,14 +362,17 @@ test('project#vs', t => {
     const module = projectVS(uniforms);
 
     testCase.tests.forEach(c => {
-      const actual = c.func(module);
+      let actual = c.func(module);
```

Comment:
Only run this if webgl is not available?

Reply:
Done, run tests using Transform when supported, otherwise fallback to transpiler.